client/src/Components/Common/DateTimePicker/index.js

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+import { TextField } from '@mui/material';
+import moment from 'moment';
+
+export default function DateTimePicker() {
```

Comment:
```suggestion
function DateTimePicker() {
```
export component at the end of the file